x-pack/test/encrypted_saved_objects_api_integration/tests/encrypted_saved_objects_api.ts

```diff
@@ -243,19 +345,37 @@ export default function({ getService }: FtrProviderContext) {
   }
 
   describe('encrypted saved objects API', () => {
+    // To workaround bug in SO bulk operations that return "raw" IDs for SO that were created
```

Comment:
__note:__ this is the related bug https://github.com/elastic/kibana/issues/65088